tests/conftest.py

```diff
@@ -28,6 +38,10 @@ def pytest_configure(config):
         plugin.mypy_argv.append('--ignore-missing-imports')
 
 
+def pytest_sessionstart(session):
+    os.environ.update(_FALCON_TEST_ENV)
```

Comment:
I think `setdefault`  would be better, since it allows controlling the variables from the extern

Reply:
sure